stripe/connectCallback.js

```diff
@@ -38,8 +41,19 @@ export const main = async event => {
 
   /** @type {string} error */
   const error = event.queryStringParameters.error;
+  const secretName = `${env.STAGE}/stripe`;
+  /** @type {typings.StripeSecrets} */
+  let stripeSecrets;
+  try {
+    console.log(`Get secret with name ${secretName}`);
+    stripeSecrets = await secretManagerLib.getSecrets(secretName);
+  } catch (e) {
+    console.error(e);
+    return failure({ status: false });
+  }
```

Comment:
I wonder if it would be a bit more convenient to extract the method to get `stripe`.
smg like 

```
function getStripe(){
  const secretName = `${env.STAGE}/stripe`;
  /** @type {typings.StripeSecrets} */
  let stripeSecrets;
  try {
    console.log(`Get secret with name ${secretName}`);
    stripeSecrets = await secretManagerLib.getSecrets(secretName);
  } catch (e) {
    console.error(e);
    return failure({ status: false });
  }

  console.log('Initialize Stripe');	  console.log('Initialize Stripe');
  const stripe = Stripe(env.STRIPE_API_SECRET_KEY);	  
return Stripe(stripeSecrets.API_SECRET_KEY);
}
```

Reply:
Probably this way we will not need a separate file for typing since `stripeSecrets` would be used just in one place

Reply:
I like the idea